modules/dnn/test/test_layers.cpp

```diff
@@ -618,6 +618,1331 @@ TEST(Layer_LSTM_Test_Accuracy_with_, HiddenParams)
+        output_shape.erase(output_shape.begin());
+    }
+
+    // generate random positeve value from 1 to 10
+    float inp_value = 1 + static_cast <float> (rand()) / (static_cast <float> (RAND_MAX/(10-1)));
+    cv::Mat input = cv::Mat(input_shape, CV_32F, inp_value);
+    cv::Mat output_ref;
+
+    output_ref = cv::Mat(output_shape, CV_32F, 1.0);
+    std::vector<Mat> inputs{input};
+    std::vector<Mat> outputs;
+
+    runLayer(layer, inputs, outputs);
+    ASSERT_EQ(shape(output_ref), shape(outputs[0]));
+    normAssert(output_ref, outputs[0]);
+}
+INSTANTIATE_TEST_CASE_P(/*nothing*/, Layer_ReLU6_1d_Test,
+/*operation*/           Values(0, 1));
+
+
+typedef testing::TestWithParam<tuple<int>> Layer_Clip_1d_Test;
+TEST_P(Layer_Clip_1d_Test, Accuracy)
+{
+    int batch_size = get<0>(GetParam());
+    LayerParams lp;
+
+    lp.type = "Clip";
+    lp.name = "clipLayer";
+    lp.set("min_value", 0.0);
+    lp.set("max_value", 1.0);
+    Ptr<ReLU6Layer> layer = ReLU6Layer::create(lp);
+
+    std::vector<int> input_shape = {1, 3};
+    std::vector<int> output_shape = {1, 3};
+    if (batch_size == 0) {
+        input_shape.erase(input_shape.begin());
+        output_shape.erase(output_shape.begin());
+    }
+
+    float inp_value = 1 + static_cast <float> (rand()) / (static_cast <float> (RAND_MAX/(10-1)));
+    cv::Mat input = cv::Mat(input_shape, CV_32F, inp_value);
+    cv::Mat output_ref = cv::Mat(output_shape, CV_32F, 1.0);
+    std::vector<Mat> inputs{input};
+    std::vector<Mat> outputs;
+
+    runLayer(layer, inputs, outputs);
+    ASSERT_EQ(shape(output_ref), shape(outputs[0]));
+    normAssert(output_ref, outputs[0]);
+}
+INSTANTIATE_TEST_CASE_P(/*nothing*/, Layer_Clip_1d_Test, Values(0, 1));
+
+typedef testing::TestWithParam<tuple<int>> Layer_ReLU_1d_Test;
+TEST_P(Layer_ReLU_1d_Test, Accuracy)
+{
+    int batch_size = get<0>(GetParam());
+    LayerParams lp;
+
+    lp.type = "ReLU";
+    lp.name = "reluLayer";
+    lp.set("negative_slope", 0.0);
+    Ptr<ReLULayer> layer = ReLULayer::create(lp);
+
+    std::vector<int> input_shape = {1, 3};
+    std::vector<int> output_shape = {1, 3};
+    if (batch_size == 0) {
+        input_shape.erase(input_shape.begin());
+        output_shape.erase(output_shape.begin());
+    }
+
+    float inp_value = 1 + static_cast <float> (rand()) / (static_cast <float> (RAND_MAX/(10-1)));
+    cv::Mat input = cv::Mat(input_shape, CV_32F, inp_value);
+    cv::Mat output_ref = cv::Mat(output_shape, CV_32F, inp_value);
+    std::vector<Mat> inputs{input};
+    std::vector<Mat> outputs;
+
+    runLayer(layer, inputs, outputs);
+    ASSERT_EQ(shape(output_ref), shape(outputs[0]));
+    normAssert(output_ref, outputs[0]);
+}
+INSTANTIATE_TEST_CASE_P(/*nothing*/, Layer_ReLU_1d_Test, Values(0, 1));
+
+typedef testing::TestWithParam<tuple<int>> Layer_GeLU_1d_Test;
+TEST_P(Layer_GeLU_1d_Test, Accuracy)
+{
+    int batch_size = get<0>(GetParam());
+    LayerParams lp;
+
+    lp.type = "GeLU";
```

Comment:
It should be "Gelu" instead (lower-case l and u). It was fine because you create a layer directly instead of creating a net.